.github/workflows/pull.yml

```diff
@@ -449,6 +449,26 @@ jobs:
         ]}
     secrets: inherit
 
+  unstable-linux-focal-cuda12_4-py3_10-gcc9-sm89-build-xfail:
+    # A version of the build that sets a larger number of jobs for a build.  May
+    # OOM
+    name: unstable-linux-focal-cuda12.4-py3.10-gcc9-sm89-xfail
+    uses: ./.github/workflows/_linux-build.yml
+    needs: get-label-type
+    with:
+      runner_prefix: "${{ needs.get-label-type.outputs.label-type }}"
+      build-environment: linux-focal-cuda12.4-py3.10-gcc9-sm89
+      docker-image-name: pytorch-linux-focal-cuda12.4-cudnn9-py3-gcc9
+      cuda-arch-list: 8.9
+      max-jobs: 4
```

Comment:
Seems like this is the only job preserving the old behavior?

Reply:
The old behavior is that max_jobs gets set to nprocs - 2 = 8 in the docker image, and then the build bash script sets it to 2 later on

Reply:
If I read this correctly, this job now fixes the number or parallel build process to 4 and bypasses all the MAX_JOBS calculation